### ISSUE TITLE
Hover doesn't work on parallel edges

### ISSUE THREAD

**Issue by iletunic** [jacomyal/sigma.js#630]
Edges do get highlighted, but when pointer is at a different place, not over the actual edge.

Is it possible that isPointOnBezierCurve function needs to be modified?

**Comment by sheymann** [comment]
Yes, isPointOnBezierCurve is bound to sigma.canvas.edges.curve.js and doesn't support various control points.

**Comment by bentwonk** [comment]
Has this been addressed anywhere, I could really use selectable parallel edges.

**Comment by sheymann** [comment]
Hello, it works in this fork: https://github.com/Linkurious/linkurious.js

Demo: https://rawgit.com/Linkurious/linkurious.js/develop/examples/autocurve-parallel-edges.html

**Comment by bferi** [comment]
Can someone tell why is this not included in this repository? I'm just curious.
